frontend/src/app/modules/auth/register-form/register-form.component.html

```diff
@@ -2,46 +2,90 @@
             <input [type]="hidePass ? 'password' : 'text'" matInput minlength="8" placeholder="password"
-                formControlName="password" required>
+                formControlName="password">
             <mat-icon class="password-icon" matSuffix (click)="hidePass = !hidePass">{{hidePass ? 'visibility' :
                 'visibility_off'}}
             </mat-icon>
         </mat-form-field>
+        <div class="errors">
+            <small *ngIf=" registerForm.controls['password'].touched && registerForm.controls['password'].errors && registerForm.controls['password'].errors['required']">
+                Password is a required property
+            </small>
+            <small *ngIf=" registerForm.controls['password'].touched && registerForm.controls['password'].errors && 
+            ( registerForm.controls['password'].errors['pattern'] || 
+              registerForm.controls['password'].errors['minlength'] ||
+              registerForm.controls['password'].errors['maxlength'] )">
+                At least 1 digit, at least 1 letter, 8-20 characters
+            </small>
+        </div>
+
         <p class="input-label">Repeat Password</p>
-        <mat-form-field class="input-form" appearance="fill" color="warn" floatLabel="always" hideRequiredMarker="true">
-            <input [type]="hideConfirmPass ? 'password' : 'text'" matInput placeholder="password"
-                formControlName="confirmPassword" required>
-            <mat-icon class="password-icon" matSuffix (click)="hideConfirmPass = !hideConfirmPass">{{hideConfirmPass ?
-                'visibility' :
+        <mat-form-field class="input-form" appearance="fill" hideRequiredMarker="true">
+            <input [type]="hideConfirmPass ? 'password' : 'text'" matInput minlength="8" placeholder="confirm password"
+                formControlName="confirmPassword">
+            <mat-icon class="password-icon" matSuffix (click)="hideConfirmPass = !hideConfirmPass">{{hideConfirmPass ? 'visibility' :
                 'visibility_off'}}
             </mat-icon>
         </mat-form-field>
+        <div class="errors">
+            <small *ngIf=" registerForm.controls['confirmPassword'].touched && registerForm.controls['confirmPassword'].errors && registerForm.controls['confirmPassword'].errors['required']">
+                Confirm password is a required property
+            </small>
+            <small *ngIf=" registerForm.controls['confirmPassword'].touched && registerForm.controls['confirmPassword'].errors && registerForm.controls['confirmPassword'].errors['passwordMatchValidator']">
+                Passwords do not match
+            </small>
+        </div>
+
         <div class="registation-buttons">
-            <button class="submit-button" mat-button type="submit" [disabled]="!registerForm.valid">Sign
-                Up</button>
+            <button class="submit-button" mat-button type="submit" [disabled]="!registerForm.valid">
```

Comment:
It doesn't look a lot like disabled. 
Can you play with styles a bit?
![image](https://user-images.githubusercontent.com/20342130/154927243-2e38cf60-3dfd-4f3a-b94b-726ad424cdd8.png)